.env.example

```
@@ -222,3 +222,25 @@ RELAYER_TOKENS='["0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2", "0xA0b86991c6218b
 #    },
 #  },
 #}'
+
+################################################################################
+########################### Testing Configuration ##############################
+################################################################################
+
+# Note: This section is intended for advanced users only. It ONLY serves to aid
+#       developers in testing the relayer bot. It is NOT intended for any relayer 
+#       or dataworker operator to use in production. It's recommended to consult 
+#       the #relayers channel within the Across Discord server before making any 
+#       changes to this section. See https://discord.across.to.
+#
+# Note: PLEASE DO NOT USE THIS SECTION IN PRODUCTION. IT IS FOR TESTING ONLY.
+
+# Used to include an arbitrary chain ID to the Across Config Store for testing
+# new chain additions. Including the block number and the chain ID will cause
+# the relayer to include the chain ID in the Config Store. This is useful for
+# testing new chains before they are officially supported by the relayer.
+# Note: This logic does apply sanitizations so it is not possible to inject
+#       a new change prior to the most previous chain ID update within the
+#       Config Store.
+INJECT_CHAIN_ID_INCLUSION='{"blockNumber":17876743,"chainId":8453}'
```

Default leave commented out?
```suggestion
#INJECT_CHAIN_ID_INCLUSION='{"blockNumber":17876743,"chainId":8453}'
```

---

Great call